doc/src/Build_settings.rst

```diff
@@ -76,6 +82,8 @@ LAMMPS can use them if they are available on your system.
          -D MKL_INCLUDE_DIR=path     # ditto for Intel MKL library
          -D FFT_MKL_THREADS=on       # enable using threaded FFTs with MKL libraries
          -D MKL_LIBRARY=path         # path to MKL libraries
+         -D Heffte_ROOT=path         # path to an existing heFFTe installation
+
```

Comment:
Need to document how to use Heffte with Traditional Make as well, i.e. add the flag below.